callstack/react-native-testing-library
# docs: cookbook shoutout ready

### Summary

Prep. Cookbook App/Docs to be shoutout ready for announcement via social channels

### Test plan

1. Run Cookbook app by following it's updated readme.
2. Ensure readability, cohesion in story and no dead links.
